linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/var_password_pam_remember_control_flag.var

```diff
@@ -2,7 +2,10 @@ documentation_complete: true
 
 title: 'PAM pwhistory remember - control flag'
 
-description: 'Specify the control flag required for password remember requirement.'
+description: |-
+    'Specify the control flag required for password remember requirement. If multiple
+    values are allowed write them separated by commas as in "required,requisite",
+    for remediations the first value will be taken'
```

Comment:
I had to read the description twice. Maybe we could subtly improve the reading. For example:
```
    'Specify the control flag required for password remember requirement. If multiple
    values are allowed, write them separated by commas as in "required,requisite".
    In case of multiple values, the first value will be taken for remediations.'
```